common/metrics/defs.go

```diff
@@ -1437,6 +1437,7 @@ const (
 
 	MatchingClientForwardedCounter
 	MatchingClientInvalidTaskListName
+	CadenceErrIncompleteHistoryCounter
```

Comment:
i think you can put it on line 1370? near CadenceErrClientVersionNotSupportedCounter